kernel/page_attribute_table/src/lib.rs

```diff
@@ -60,6 +60,9 @@ pub static FIXED_PAT: PageAttributeTable = PageAttributeTable::from_bytes([
     MemoryCachingType::UncachedMinus  as u8, // 7: 0b111
 ]);
 
+pub enum PageAttributeTableError {
+    Unsupported,
+}
```

Comment:
might as well make this an empty struct or enum called `PatUnsupported`, since there's only one variant